src/extract_sf.c

```diff
@@ -268,7 +278,7 @@ int run(char *in_filename, bool use_fmt, int nevn, int run_no) {
     }
 
     // Write results to file.
-    FILE *t_out = fopen(Form("../data/sf_params_%06d.root", run_no), "w");
+    FILE *t_out = fopen(Form("../root_io/sf_params_%06d.root", run_no), "w");
```

Comment:
Also this. sampling fraction parameters were saved to `data` instead of `root_io` for a reason. Why is this changed? Additionally, this suffers from the same effect as the sf_study filename since the run_no is invalid.

Reply:
Reverted to old directory convention.